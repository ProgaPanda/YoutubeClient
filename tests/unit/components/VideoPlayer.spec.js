import { shallowMount } from '@vue/test-utils';
import VideoPlayer from '@/components/VideoPlayer.component.vue';

describe('VideoPlayer', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(VideoPlayer, {
      propsData: {
        title: 'video title',
        views: '1000',
        reactions: { likes: 10, dislikes: 10 },
      },
    });

    expect(wrapper.element).toMatchSnapshot();
  });
});
