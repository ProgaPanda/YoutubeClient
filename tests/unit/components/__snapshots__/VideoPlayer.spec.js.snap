// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer renders correctly 1`] = `
<div
  class="video-player"
>
  <!---->
   
  <div
    class="video-player__video-container"
  >
    <iframe
      allowfullscreen="allowfullscreen"
      class="video"
      src="https://www.youtube.com/embed/undefined"
    />
  </div>
   
  <div
    class="video-player__info"
  >
    <h3
      class="video-player__info__title"
    >
      video title
    </h3>
     
    <div
      class="video-player__info__meta"
    >
      <p
        class="video-player__info__meta__stats"
      >
        
        1,000 views • May 7, 2020
      
      </p>
       
      <p
        class="video-player__info__meta__reactions"
      >
        
        10 
        <span
          class="likes-wording"
        >
          likes
        </span>
         •
        10
        
        <span
          class="dislikes-wording"
        >
          dislikes
        </span>
      </p>
    </div>
  </div>
</div>
`;
